api/v1alpha1/ingressnodefirewallconfig_types.go

```diff
@@ -21,27 +21,26 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// IngressNodeFirewallConfigSpec defines the desired state of IngressNodeFirewallConfig
+// IngressNodeFirewallConfigSpec defines the desired state of IngressNodeFirewallConfig.
 type IngressNodeFirewallConfigSpec struct {
-	// Nodes where the daemonset will be running on.
+	// nodeSelector is used to select which Nodes the ingress node firewall DaemonSet will be run on.
+	// + This field is using map[string]string instead of metav1.LabelSelector because it matches the daemonset NodeSelector template. Thus one cannot benefit from additional match criteria that metav1.LabelSelector provides.
 	// +optional
 	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
-	// tolerations is a list of tolerations applied to ingress node firewall controller
-	// deployments.
+	// tolerations is a list of tolerations applied to ingress node firewall controller.
```

Comment:
// tolerations is a list of tolerations applied to the ingress node firewall controller
// DaemonSet.